src/heuristics.jl

```diff
@@ -100,15 +100,17 @@ function get_value(diagram, S_active, j, s_j, s_Ij, EU)
     return EU, S_active_new
 end
 
-function set_MIP_start(diagram, Z, S_active, z_z, x_s)
+function set_MIP_start(diagram, Z, S_active, z_z; x_s)
     for (k,j) in enumerate(Z.D)
         for s_Ij in paths(get_values(diagram.S)[Z.I_d[k]])
                 set_start_value(z_z[k][s_Ij..., Z.Z_d[k](s_Ij)], 1)
         end
     end
 
-    for s in S_active
-        set_start_value(x_s[s], 1)
+    if x_s != nothing
```

Comment:
Does this still set the MIP start correctly, can HiGHS use the start values for just the z-variables?

Reply:
Seems to be working correctly, the model output gives "MIP start solution is feasible, objective value is x", which is the correct solution. However, spu running times seem to increase exponentially at least for the pig breeding problem.